merger_sort.c

```diff
@@ -0,0 +1,98 @@
+#include <stdio.h>
+
+#define SIZE 8
+
+int temp[SIZE] = {0};
+
+void merge (int array[], int start_1, int end_1, int start_2, int end_2)
```

Comment:
Personally I'd opt for two arrays, instead of start_1, start_2, end_1, end_2. Something along the lines of
void merge (int left[], int leftSz, int right[] int rightSz)